data/docs_markdown/Extending_Your_Cluster_and_Pay.md

```
@@ -0,0 +1,29 @@
+# Extending Your Cluster and Pay
+## [](https://docs.io.net/docs/extending-your-cluster-and-pay#table-of-contents)
+  * [Extending Your Cluster and Paying with $IO Coin](https://docs.io.net/docs/extending-your-cluster-and-pay#extending-your-cluster-and-paying-with-io-coin)
+  * [Etending Your Cluster and Paying with USDC](https://docs.io.net/docs/extending-your-cluster-and-pay#extending-your-cluster-and-paying-with-usdc)
+
+
+### [](https://docs.io.net/docs/extending-your-cluster-and-pay#extending-your-cluster-and-paying-with-io-coin)
+To expand your cluster, you need to click **Expand Cluster** in the active cluster
+![](https://files.readme.io/d31ad673043854930306b8e96695f19eedd3481e437d06fca8023755056fcce2-Extending_Cluster_1.png)
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)

(MD045, no-alt-text)